pages/recordbag.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+import RecordFile from '../components/RecordFile';
+
+export default function RecordBag({ selectionState, onHandleChange }) {
+  const selection = selectionState;
+
+  return (
+    <>
+      <Heading>
+        <h1>RecordBag</h1>
+      </Heading>
+      <Section>
+        <Text>Your selected records:</Text>
+        <Selection>
+          {selection.map((file) => (
+            <RecordFile
+              key={file.CatalogId}
+              record={file}
+              onHandleChange={onHandleChange}
+            />
+          ))}
+        </Selection>
+      </Section>
+    </>
+  );
+}
+
+const Selection = styled.ul`
+  list-style: none;
+  position: absolut;
```

Comment:
```suggestion
  position: absolute;
```
Still really needed?

Reply:
It is needed to avoid that the rendered cards have a width of 100%. But fine tuning of the styling will have to happen later anyways.